setup.py

```diff
@@ -65,103 +77,91 @@ def run(self):
-                'blockchain/eth/contract_registry/mainnet/*',
-                'blockchain/eth/contract_registry/cassandra/*',
-                'blockchain/eth/contract_registry/gemini/*',
-                'blockchain/eth/contract_registry/frances/*',
-                'blockchain/eth/contract_registry/miranda/*',
-                'blockchain/eth/sol/source/contracts/*',
-                'blockchain/eth/sol/source/contracts/lib/*',
-                'blockchain/eth/sol/source/contracts/proxy/*',
-                'blockchain/eth/sol/source/zeppelin/math/*',
-                'blockchain/eth/sol/source/zeppelin/utils/*',
-                'blockchain/eth/sol/source/zeppelin/ownership/*',
-                'blockchain/eth/sol/source/zeppelin/token/ERC20/*',
-                'blockchain/eth/sol/source/aragon/contracts/*',
-                'blockchain/eth/sol/source/aragon/interfaces/*',
-                ]
-
-setup(name=ABOUT['__title__'],
-      url=ABOUT['__url__'],
-      version=ABOUT['__version__'],
-      author=ABOUT['__author__'],
-      author_email=ABOUT['__email__'],
-      description=ABOUT['__summary__'],
-      license=ABOUT['__license__'],
-      long_description=long_description,
-      long_description_content_type="text/markdown",
-
-      # Setup
-      python_requires='>=3',
-      setup_requires=['pytest-runner'],  # required for `setup.py test`
-      tests_require=TESTS_REQUIRE,
-      install_requires=INSTALL_REQUIRES,
-      extras_require=EXTRAS_REQUIRE,
-
-      # Package Data
-      packages=find_packages(exclude=["tests"]),
-      package_data={PACKAGE_NAME: PACKAGE_DATA},
-      include_package_data=True,
-
-      # Entry Points
-      entry_points={'console_scripts': [
-          f'{PACKAGE_NAME} = {PACKAGE_NAME}.cli.main:nucypher_cli',
-          f'{PACKAGE_NAME}-deploy = {PACKAGE_NAME}.cli.commands.deploy:deploy',
-      ]},
-      cmdclass={'verify': VerifyVersionCommand},
-
-      # Metadata
-      classifiers=[
-          "Development Status :: 3 - Alpha",
-          "Intended Audience :: Developers",
-          "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)",
-          "Natural Language :: English",
-          "Operating System :: OS Independent",
-          "Programming Language :: Python",
-          "Programming Language :: Python :: 3 :: Only",
-          "Programming Language :: Python :: 3.6",
-          "Programming Language :: Python :: 3.7",
-          "Programming Language :: Python :: 3.8",
-          "Topic :: Security"
-      ])
+URSULA_REQUIRES = ['prometheus_client']
+
+EXTRAS = {
+    'ursula': URSULA_REQUIRES,
+    'docs': DOCS_REQUIRE,
+    'dev': DEV_REQUIRES + DOCS_REQUIRE,
+    'benchmark': DEV_REQUIRES + BENCHMARK_REQUIRES
+}
+
+
+setup(
+
+    # Requirements
+    python_requires='>=3',
+    setup_requires=['pytest-runner', 'setuptools-markdown'],
+    install_requires=INSTALL_REQUIRES,
+    extras_require=EXTRAS,
```

Comment:
Previously there was a kwarg `tests_require`. Was dropping this intentional?

Reply:
Yeah, intentional.  The `test_require` kwarg is part of support for running tests through `python setup.py test` which we will not support.

Reply:
:heavy_check_mark:  (also removed pytest-runner from `setup_requires`)